### PR TITLE
Issue19 create tables

### PR DESCRIPTION
**Pull request by SreepriyaV**
### Assignee Tasks

- [ ] added unit tests (or none needed)
- [ ] written relevant docs (or none needed)
- [ ] referenced any relevant issues (or none exist)

### Guidelines

Please add a description of this Pull Request's motivation, scope, outstanding issues or potential alternatives, reasoning behind the current solution, and any other relevant information for posterity.

---

*Your PR Notes Here*

### CODE REVIEW

**Comment by gracemgately** [comment]
I see there's an extra boilermaker-master folder with duplicates of all the public and server files?

File: server/db/models/index.js [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-/**
- * If we had any associations to make, this would be a great place to put them!
- * ex. if we had another model called BlogPost, we might say:
- *
- *    BlogPost.belongsTo(User)
- */
+Orders.belongsTo(User, { as: "user" });
+User.hasMany(Orders);
+Reviews.belongsTo(User, { as: "user" });
+Reviews.belongsTo(Cats, { as: "cat" });
```
Comment (intersim) [inline server/db/models/index.js]: Why are you doing this aliasing?

File: server/db/models/orders.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+const Orders = db.define("orders", {
+  cart: {
+    type: Sequelize.ARRAY(Sequelize.STRING),
+   
+  },
+  status: {
+    type: Sequelize.STRING,
+    validate:{
+        isIn:[['Created',  'Processing', 'Cancelled', 'Completed']]
+    }
```
Comment (intersim) [inline server/db/models/orders.js]: This seems like a good place to use an ENUM data type.

File: server/db/models/cats.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  },
+  color: {
+    type: Sequelize.STRING
+  },
+  hairLength: {
+    type: Sequelize.STRING
+  },
+  profession: {
+    type: Sequelize.STRING
+  },
```
Comment (intersim) [inline server/db/models/cats.js]: Cats have a profession??

File: package.json [hunk 4]
```diff
@@ -13,15 +13,16 @@
     "lint": "eslint ./ --ignore-path .gitignore",
     "postinstall": "touch secrets.js",
     "seed": "node script/seed.js",
-    "start": "node server",
+    "start": " webpack -w & nodemon server",
```
Comment (intersim) [inline package.json]: You can certainly keep this, but our start-dev script is already basically doing these steps... and we'll eventually want to change our start script back to just `node server`, as that's what we'll want Heroku to do to start our dyno (virtual machine that will run our server). I recommend keeping the start script as-is, and getting in the habit of using the `start-dev` script while you're developing your app.

File: server/db/models/cats.js (hunk 3 above)
Comment (cyng24) [reply server/db/models/cats.js]: Yes. Our cats are professionals. :)

File: server/index.js [hunk 5]
```diff
@@ -91,7 +91,7 @@ const startListening = () => {
   require('./socket')(io)
 }
 
-const syncDb = () => db.sync()
+const syncDb = () => db.sync({force: true})
```
Comment (intersim) [inline server/index.js]: This is going to drop everything in your database whenever you start your server over again! Are you sure you want this?

Comment (SreepriyaV) [reply server/index.js]: Oops. I removed that.

**Pull request merged**